From 0358e38f8dc103e46066c0049c4dc016aecd373a Mon Sep 17 00:00:00 2001
From: Zhaoyilunnn <dilved1014@gmail.com>
Date: Tue, 16 Apr 2024 16:15:55 +0800
Subject: [PATCH] fix: min not defined

---
 qisa-as/qisa_driver.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qisa-as/qisa_driver.cpp b/qisa-as/qisa_driver.cpp
index 7a06fe2..e1e53e1 100644
--- a/qisa-as/qisa_driver.cpp
+++ b/qisa-as/qisa_driver.cpp
@@ -3,6 +3,7 @@
 #include <iomanip>
 #include <iostream>
 #include <cstring>
+#include <limits>
 
 #include "qisa_driver.h"
 #include "qisa_version.h"
-- 
2.34.1

